plugins/pulsar-records-storage/src/test/java/com/github/bsideup/liiklus/pulsar/PulsarRecordsStorageTest.java

```diff
@@ -79,6 +80,12 @@ public int getNumberOfPartitions() {
         return NUM_OF_PARTITIONS;
     }
 
+    @Override
+    @Test
+    @Disabled("#180 - Pulsar should fix the way seek works, not disconnecting consumers (apache/pulsar/pull/5022)")
```

Comment:
Could you please make it disabled only until some date, like 1st of December or something?
So that we will start getting errors after that date if Pulsar did not release the fix yet?

Reply:
@lanwen ping :)